app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -3224,21 +3226,23 @@ class BrowserTabViewModel @Inject constructor(
         }
     }
 
-    private fun permissionsQuery(
+    private suspend fun permissionsQuery(
```

Comment:
You’re launching a new coroutine inside this function, so it doesn’t need to be a suspended function. Also, the call to `permissionQuery()` is from within _another_ coroutine launcher (which is also the same dispatcher as here). I think you want to use one or the other. WDYT?

Reply:
Fixed, I missed that on my second commit, thanks!